gamemodes/core/admin/admin_cmd_lvl1.inc

```diff
@@ -228,4 +228,14 @@ ACMD:[1]afix(playerid, params[])
 	if(!IsPlayerInAnyVehicle(playerid)) return SendErrorMsg(playerid, "You're not in any vehicle");
 	RepairVehicle(GetPlayerVehicleID(playerid));
 	return 1;
+}
+
+ACMD:[1]asay(playerid, params[])
```

Comment:
I removed /asay because we have # for admin chat, this is a bit redundant.

Reply:
This is like a /announce in chat for admins